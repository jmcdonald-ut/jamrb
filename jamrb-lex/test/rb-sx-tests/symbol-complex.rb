:"foO$23#{reed}"
:"HERE WE GO!"
