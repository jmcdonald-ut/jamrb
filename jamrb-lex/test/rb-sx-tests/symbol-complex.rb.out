((Token ((1 0) on_symbeg ":\""))
 (Token ((1 2) on_tstring_content "foO$23"))
 (Token ((1 8) on_embexpr_beg "\\#{"))
 (Token ((1 10) on_ident "reed"))
 (Token ((1 14) on_embexpr_end "}"))
 (Token ((1 15) on_tstring_end "\""))
 (Token ((1 16) on_nl "\n"))
 (Token ((2 0) on_symbeg ":\""))
 (Token ((2 2) on_tstring_content "HERE WE GO!"))
 (Token ((2 13) on_tstring_end "\""))
 (Token ((2 14) on_nl "\n")))
